### PR TITLE
Devoncarew embeddings

### PR DESCRIPTION
**Pull request by devoncarew**
- re-layout the `embed-dart.html` embedding (the two other embeddings are largely unchanged)
- whitespace and style changes to the html and css code
- move more style and layout behavior into the embed-dart.html file and out of the components file
- use divs to help with the positioning and layout
- prefer flexbox to width / height 100%
- rename some custom elements to have more descriptive names

screen shot 2015-06-19 at 12 10 31 am

http://dev.dart-pad.appspot.com/embeddings_demo.html

@Georgehe4 for review; let's talk in person as well - I want to make sure that these changes make sense

### CODE REVIEW

File: web/embed_components.html [hunk 1]
```diff
@@ -35,22 +35,23 @@
...
+      font-weight: 400;
+    }
   </style>
   <template>
     <div>
-      <paper-tabs class="tabs" selected="0" style = "position:relative" class="bottom fit vertical" style="top: inherit">
-          <span class="flex"></span>
-          <paper-icon-button id = "refresh" icon="refresh"></paper-icon-button>
-          <paper-icon-button id = "launch" icon="launch"></paper-icon-button>
+      <paper-tabs class="tabs" selected="0" style="position:relative" class="bottom fit vertical" style="top: inherit">
```
Comment (Georgehe4) [inline web/embed_components.html]: Remove styling inside this paper-tab.

File: web/embed_components.html [hunk 2]
```diff
@@ -2,28 +2,28 @@
...
-        <paper-tab name="css">CSS</paper-tab>
-      </paper-tabs>
-    <paper-progress id="edit-progress"  class="bottom fit" hidden
-        style="top: inherit"></paper-progress>
+    <paper-tabs id="selector-tabs" class ="tabs" selected="0" class="bottom fit vertical">
+      <paper-tab name="dart">Dart</paper-tab>
+      <paper-tab name="html">HTML</paper-tab>
+      <paper-tab name="css">CSS</paper-tab>
+    </paper-tabs>
+    <paper-progress id="edit-progress"  class="bottom fit" hidden style="top: inherit"></paper-progress>
```
Comment (Georgehe4) [inline web/embed_components.html]: Remove styling inside html

File: web/embed_components.html [hunk 3]
```diff
@@ -133,28 +136,28 @@
...
-        <paper-progress id="edit-progress"  class="bottom fit" hidden
-            style="top: inherit"></paper-progress>
+    <div class="layout horizontal flex">
+      <div class="layout horizontal flex-6">
+        <paper-tabs id="selector-tabs" class ="tabs" selected="0" class="flex">
+          <paper-tab name="dart">Dart</paper-tab>
+          <paper-tab name="html">HTML</paper-tab>
+          <paper-tab name="css">CSS</paper-tab>
+        </paper-tabs>
+        <paper-progress id="edit-progress"  class="bottom fit" hidden style="top: inherit"></paper-progress>
```
Comment (Georgehe4) [inline web/embed_components.html]: Style

File: web/embed_components.html [hunk 4]
```diff
@@ -133,28 +136,28 @@
...
       </div>
-      <div class = "layout horizontal flex-4"> 
+      <div class="layout horizontal flex-4">
         <paper-tabs class="tabs" selected="0" class="flex" style="top: inherit">
```
Comment (Georgehe4) [inline web/embed_components.html]: Style

File: web/embed_components.html [hunk 5]
```diff
@@ -207,26 +215,29 @@
...
-     height: 100%;
-   }
-   .maxWidth {
-     width: 100%;
-   }
+    #frameContainer {
+      position: fixed;
+    }
+
+    #frame {
```
Comment (Georgehe4) [inline web/embed_components.html]: may want to fix 100% elements

**Comment by Georgehe4** [comment]
lgtm

File: web/embed_components.html (hunk 1 above)
Comment (devoncarew) [inline web/embed_components.html]: done

File: web/embed_components.html (hunk 2 above)
Comment (devoncarew) [inline web/embed_components.html]: will do, but I'm not familiar with what this is here for

File: web/embed_components.html (hunk 3 above)
Comment (devoncarew) [inline web/embed_components.html]: done

File: web/embed_components.html (hunk 4 above)
Comment (devoncarew) [inline web/embed_components.html]: done

File: web/embed_components.html (hunk 5 above)
Comment (devoncarew) [inline web/embed_components.html]: let's tackle this in another PR

**Comment by devoncarew** [comment]
updated - merging

**Pull request merged**